src/test/java/com/mateacademy/sql/ConnectorUtilTest.java

```diff
@@ -0,0 +1,21 @@
+package com.mateacademy.sql;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class ConnectorUtilTest {
```

Comment:
Really it is not a test.

Reply:
Its test for connection, i can delete this class.
Because i dont know how to test this program